src/components/StatisticCard/StatisticCard.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import cx from 'classnames';
+
+import RichText from 'RichText';
+import Link from 'Link';
+
+type StatisticCardProps = {
+  className?: string;
+  description?: string;
+  href?: string;
+  linkTitle?: string;
+  title?: string;
+  titleAs?: 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+};
+
+export const StatisticCard = ({
+  className,
+  description,
+  href,
+  linkTitle,
+  title,
+  titleAs = 'h3'
+}: StatisticCardProps) => {
+  const TitleElement = titleAs;
+  const classNames = cx('cc-statistic-card', {
+    [className]: className
+  });
+
+  return (
+    <article className={classNames}>
+      <TitleElement className="cc-statistic-card__title" itemProp="name">
+        {title}
+      </TitleElement>
+      {description && (
+        <RichText
+          className="cc-statistic-card__description"
+          itemProp="abstract"
+        >
+          {description}
+        </RichText>
+      )}
+      {linkTitle && (
+        <Link className="cc-statistic-card__link" to={href}>
```

Comment:
Looking at the creative this CTA and arrow looks very much like the one from FullWidthPromo (and I suspect this arrow may be animated like it too). Maybe double check with Jason.

If it is you can reuse the [Button markup](https://github.com/wellcometrust/corporate-components/blob/78731e38d004152eb941e1879521b2056fb2cc66/src/components/FullWidthPromo/FullWidthPromo.tsx#L96) from that component.

Reply:
Having it consistency makes sense. Will change it.